config.py

```diff
@@ -54,10 +54,7 @@ def main():
     config_data = yaml.safe_load(open('sysconfigs/' + args.machine.replace('admin-', '') + '/config.yaml', 'r'))
 
     if not args.pckgidir:
-        if 'admin' in args.machine:
-            args.pckgidir = '/project/g110'
-        else:
-            args.pckgidir = '$SCRATCH'
+        args.pckgidir = '$SCRATCH'
```

Comment:
the problem with this version is that now we will have the installation on scratch. I think we need to use a different variable for install and stages - I 'll try to come up with another solution